cmd/ctr/app/main.go

```diff
@@ -100,6 +106,10 @@ containerd CLI
 			Value:  namespaces.Default,
 			EnvVar: namespaces.NamespaceEnvVar,
 		},
+		cli.BoolFlag{
+			Name:  "opentelemetry",
```

Comment:
Right now the flag is only used to initialize OpenTelemetry package and the configuration is mostly coming from https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/. It is odd, but adding multiple flags just for OpenTelemetry is also odd.

Reply:
Can we add the exporter endpoint address as another optional field?

Reply:
wdyt about calling the flag `tracing` instead of `opentelemetry`. And the endpoint as `tracing-endpoint`. Opentelemetry in itself encompasses a number of things besides tracing and we do not necessary need to expose what tracing framework is implemented.

Reply:
I'm fine using more generic terms. We may have -trace flag somewhere already. Let me check.

Regarding `tracing-endpoint`, config.toml is currently supporting the three parameters;

https://github.com/containerd/containerd/blob/62968d9ecc1c086b92af5098b1bb099f67573399/tracing/plugin/otlp.go#L95-L99

Should we expose all of them as flags? It looks a bit overkill to do that just for OpenTelemetry.

Reply:
IMO exposing `Endpoint` should be sufficient for ctr. Mainly because sometimes we may want to run multiple backends (jaeger, otel-collector). I am fine with not making the other options configurable

Reply:
That makes sense. I'm currently only specifying `OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318` since the default seems https://.

Reply:
How about just adding `-trace http://localhost/...` to enable OpenTelemetry and configure the endpoint?

Reply:
FYI, we have `-http-trace` already.

Reply:
Yeah, sgtm. Is that a required field or can we also pass `-trace` without endpoint and it defaults to OTEL_EXPORTER_OTLP_ENDPOINT?

Reply:
We can make the endpoint optional. I will let OpenTelemetry's Go SDK decide the default value. According to https://github.com/open-telemetry/opentelemetry-go/blob/aa868d506b7e4de2f534f47b168b27b7f8824351/exporters/otlp/otlptrace/README.md#environment-variables, https:// is intentional, but it is conflicting with https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/.

Reply:
Hmm, no. I cannot make the endpoint part optional...